### ISSUE TITLE
Physics editor not present in linux

### ISSUE THREAD

**Issue by stevedupras** [UnderwaterApps/overlap2d#383]
Hello 
i have the current linux version 0.1.2 installed and seems to run great except when looking for the physics editor i cannot find it or maybe the options have been moved or changed but i am following a tutorial and they right click and select edit physics but i donot have that option??? any ideas?
thank you
steve

**Comment by azakhary** [comment]
This has moved, just click on any item, and in basic properties "components" section add physics component as component. (also polygon component and make default shape)

**Comment by stevedupras** [comment]
yes perfect thank you i stumbled upon it in the course of exploring overlap2d for my first time:) great app only  i having another issue i will make a new post...
